tests/ImageSharp.Tests/TestFile.cs

```diff
@@ -2,31 +2,56 @@
 // Copyright (c) James Jackson-South and contributors.
 // Licensed under the Apache License, Version 2.0.
 // </copyright>
+
 namespace ImageSharp.Tests
 {
-    using System;
     using System.Collections.Concurrent;
     using System.IO;
 
+    using ImageSharp.Formats;
+
+    /// <summary>
+    /// A test image file.
+    /// </summary>
     public class TestFile
     {
-        private static readonly ConcurrentDictionary<string, TestFile> cache = new ConcurrentDictionary<string, TestFile>();
-        private static readonly string FormatsDirectory = GetFormatsDirectory();
+        /// <summary>
+        /// The test file cache.
+        /// </summary>
+        private static readonly ConcurrentDictionary<string, TestFile> Cache = new ConcurrentDictionary<string, TestFile>();
 
-        private static string GetFormatsDirectory()
-        {
-          // Here for code coverage tests.
-          string directory = "TestImages/Formats/";
-          if (Directory.Exists(directory))
-          {
-              return directory;
-          }
-          return "../../../../TestImages/Formats/";
-        }
+        /// <summary>
+        /// The formats directory.
+        /// </summary>
+        private static readonly string FormatsDirectory = GetFormatsDirectory();
 
+        /// <summary>
+        /// The image.
+        /// </summary>
         private readonly Image image;
+
+        /// <summary>
+        /// The file.
+        /// </summary>
         private readonly string file;
 
+        /// <summary>
+        /// Initializes static members of the <see cref="TestFile"/> class.
+        /// </summary>
+        static TestFile()
+        {
+            // Register the individual image formats.
+            // TODO: Is this the best place to do this?
```

Comment:
Yeah, It's pretty straightforward to do this, but having an extra dll sitting on all of the other dll-s is raping the nice modularization system :D
I would rather use :bat:*Reflection*:bat: for this. (We are talking about an optional utility class!)